docs/DeveloperGuide.md

```diff
@@ -773,16 +773,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-#### Use case: Undo/redo an action
+#### Use case: Undo an action
 
 **MSS**
 
-1. placeholder
+1. User requests to undo one action by executing `undo`. 
+2. DengueHotspotTracker is restored to a state before step 1, performing an undo operation a certain number of times, according to the specified argument.
+3. A list of persons is reflecting the previous state is displayed.
 
 **Extensions**
 
-* 1a. placeholder
-    * 1a1. placeholder
+* 1a. An invalid argument is supplied.
+
+    * 1a1. DengueHotspotTracker shows an error message.
+
+    Use case ends.
```

Comment:
We need to standardise this with the rest of the DG. Right now the rest of us are looping back to the present state in your case should be before step 1.

We need to discuss how we would like to write the extensions here

Reply:
We'll look at it later